sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -1518,7 +1518,7 @@ public partial class ParameterDefinitionsValueMetadata
     public partial class ParameterValuesValue
     {
         public ParameterValuesValue() { }
-        public object Value { get { throw null; } set { } }
```

Comment:
As far as I know, this property in most cases, is a primitive type - therefore `any` is more suitable for this property.
Please consider to add a configuration to make sure this property is an `any` instead of an `anyObject`